src/DataBox/DataBox/ChangeLog.md

```diff
@@ -19,17 +19,10 @@
 -->
 
 ## Upcoming Release
-* General Availability of `Az.Databox` Module
-* Added cmdlet `Get-AzDataBoxJob`
-	- This cmdlet fetches the databox job/jobs of the user based on the filters applied like resource group name etc.
-* Added cmdlet `Get-AzDataBoxCredential`
-	- This cmdlet fetches the databox credentials of the specified databox job.
-* Added cmdlet `Stop-AzDataBoxJob`
-	- This cmdlet is used to cancel an ongoing databox job.
-* Added cmdlet `Remove-AzDataBoxJob`
-	- This cmdlet is used to delete a finished(completed/cancelled/aborted) databox job.
-* Added cmdlet `New-AzDataBoxJob`
-	- This cmdlet is used to create a new databox job.
-
+*Overview of change #1
```

Comment:
Why are you removing this?

Reply:
This is a log of all released changes, you should not remove existing changes when adding entries to this file.

Reply:
Got it. Fixed it now by restoring all previous entries.